packages/components/src/internal/components/assay/AssayImportPanels.tsx

```diff
@@ -294,14 +297,18 @@ class AssayImportPanelsBody extends Component<Props, State> {
     }
 
     onGetQueryDetailsComplete = (): void => {
-        const { assayDefinition, location } = this.props;
+        const { assayDefinition, location, loadJobSamples, loadSelections } = this.props;
         const sampleColumnData = assayDefinition.getSampleColumn();
 
         if (sampleColumnData && location) {
             // If the assay has a sample column look up at Batch, Run, or Result level then we want to retrieve
             // the currently selected samples so we can pre-populate the fields in the wizard with the selected
             // samples.
-            this.props.loadSelections(location, sampleColumnData.column).then(samples => {
+            // Additionally, we want to pre-populate the fields in the wizard with samples if the job has samples
+            // associated with it, and the assay has a sample column.
+
+            const loadGridWithSamplesFn = (location.query?.workflowJobId && isSampleManagerEnabled()) ? loadJobSamples : loadSelections;
```

Comment:
I'm not sure the check for `isSampleManagerEnabled` is  needed here since this will be true everywhere the workflow job id is attached to a URL.  I don't think it hurts anything, though.

Reply:
I agree that this isn't necessary here.